app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -29,35 +30,42 @@
 
 @Service
 public class ExperimentPageContentService {
+    final static ImmutableSet<String> EXPERIMENTS_WITH_NO_ANATOMOGRAM = ImmutableSet.of(
+            "E-CURD-10", "E-CURD-11", "E-CURD-126", "E-CURD-135",
+            "E-GEOD-86618", "E-GEOD-114530", "E-GEOD-130473",
+            "E-HCAD-8", "E-HCAD-10",
+            "E-MTAB-6308", "E-MTAB-6653", "E-MTAB-7407", "E-MTAB-9067", "E-MTAB-10662",
+            "E-ANND-1", "E-ANND-2", "E-ANND-3", "E-ANND-4", "E-ANND-5");
     private final ExperimentFileLocationService experimentFileLocationService;
     private final DataFileHub dataFileHub;
     private final TSnePlotSettingsService tsnePlotSettingsService;
     private final CellMetadataService cellMetadataService;
     private final OntologyAccessionsSearchService ontologyAccessionsSearchService;
     private final ExperimentTrader experimentTrader;
     private final CellPlotService cellPlotService;
-
-    final static ImmutableSet<String> EXPERIMENTS_WITH_NO_ANATOMOGRAM = ImmutableSet.of(
-            "E-CURD-10", "E-CURD-11", "E-CURD-126", "E-CURD-135",
-            "E-GEOD-86618", "E-GEOD-114530", "E-GEOD-130473",
-            "E-HCAD-8", "E-HCAD-10",
-            "E-MTAB-6308", "E-MTAB-6653", "E-MTAB-7407", "E-MTAB-9067", "E-MTAB-10662",
-            "E-ANND-1", "E-ANND-2", "E-ANND-3", "E-ANND-4", "E-ANND-5");
+    private final MarkerGeneService markerGeneService;
 
     public ExperimentPageContentService(ExperimentFileLocationService experimentFileLocationService,
                                         DataFileHub dataFileHub,
                                         TSnePlotSettingsService tsnePlotSettingsService,
                                         CellMetadataService cellMetadataService,
                                         OntologyAccessionsSearchService ontologyAccessionsSearchService,
                                         ExperimentTrader experimentTrader,
-                                        CellPlotService cellPlotService) {
+                                        CellPlotService cellPlotService, MarkerGeneService markerGeneService) {
         this.experimentFileLocationService = experimentFileLocationService;
         this.dataFileHub = dataFileHub;
         this.tsnePlotSettingsService = tsnePlotSettingsService;
         this.cellMetadataService = cellMetadataService;
         this.ontologyAccessionsSearchService = ontologyAccessionsSearchService;
         this.experimentTrader = experimentTrader;
         this.cellPlotService = cellPlotService;
+        this.markerGeneService = markerGeneService;
+    }
+
+    // Smart-Seq-like experiments will contain the substring “smart” in their technology types
+    private static boolean isSmartExperiment(Collection<String> technologyType) {
+        return technologyType.stream()
+                .anyMatch(type -> type.toLowerCase().matches("smart" + "-(?:.*)"));
```

Comment:
You can also use `startWith` method and extract "smart-" to a well named constant.
That way you can also remove the unnecessary comment in line 65.
Thanks

Reply:
This is fixed